README.org

```diff
@@ -0,0 +1,24 @@
+* Ring CORS
+
+  [[https://clojars.org/ring-cors][https://img.shields.io/clojars/v/ring-cors.svg]]
+  [[https://travis-ci.org/r0man/ring-cors][https://travis-ci.org/r0man/ring-cors.svg]]
+  [[http://jarkeeper.com/r0man/ring-cors][http://jarkeeper.com/r0man/ring-cors/status.svg]]
+  [[http://jarkeeper.com/r0man/ring-cors][https://jarkeeper.com/r0man/ring-cors/downloads.svg]]
+
+  [[https://github.com/ring-clojure][Ring]] middleware for [[https://en.wikipedia.org/wiki/Cross-origin_resource_sharing][Cross-Origin Resource Sharing]].
+
+** Usage
+
+   #+BEGIN_EXAMPLE
```

Comment:
Maybe not super important but you can get this syntax highlighted if you use a `SRC` block.

``` org
#+begin_src clojure
(require '[ring.middleware.cors :refer [wrap-cors]])
#+end_src
```

Hope you're well, @r0man!